phrosty/pipeline.py

```diff
@@ -131,6 +148,9 @@ def get_psf( self, ra, dec, dump_file=False ):
         #   the psf... and it's different for each type of
         #   PSF.  We need to fix that... somehow....
 
+        if self.keep_intermediate:
+            dump_file = True
```

Comment:
Why `dump_file` instead of `save_file`?

Reply:
`dia_out_dir` is mentioned in the doc string, but is not an argument to the function.  Should it refer to `self.dia_out_dir`?